src/docfx/validation/JsonSchemaValidatorExtension.cs

```diff
@@ -4,31 +4,41 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using Microsoft.Docs.Validation;
 
 namespace Microsoft.Docs.Build
 {
-    internal class JsonSchemaValidatorExtension
+    internal class JsonSchemaValidatorExtension // todo rename
```

Comment:
how about `CustomRuleProvider`?

Reply:
ValidationContext?

Reply:
Docs.Validation has had a class `ValidationContext`

Reply:
renamed to `CustomRuleProvider` after talked with yufei